dlme_airflow/dags/aims_harvester.py

```diff
@@ -0,0 +1,67 @@
+from datetime import timedelta
+from textwrap import dedent
+
+# The DAG object; we'll need this to instantiate a DAG
+from airflow import DAG
+
+# Operators; we need this to operate!
+from airflow.operators.bash import BashOperator
+from airflow.operators.python import PythonOperator
+from airflow.utils.dates import days_ago
+
+import sys
```

Comment:
I think in a follow-up PR we could either do one of the follow:

- In the Dockerfile add `/opt/dlme_airflow/` to the PYTHONPATH variable
- Allow `dlme_airflow` to be installed as a python package using a binary wheel distribution with pip, see  [documentation](https://airflow.apache.org/docs/apache-airflow/stable/modules_management.html#creating-a-package-in-python)

Reply:
Done, thanks!